providers/implementations/signature/eddsa_sig.c

```diff
@@ -526,37 +814,34 @@ static int eddsa_set_ctx_params(void *vpeddsactx, const OSSL_PARAM params[])
         char instance_name[OSSL_MAX_NAME_SIZE] = "";
         char *pinstance_name = instance_name;
 
+        if (peddsactx->instance_id_preset_flag)
+            /* Raise an error? */
```

Comment:
Indeed...

Reply:
why is this considered an error?
If I understand things right, then the operation is preset. so in theory we can just bail out with success and ignore the param.
or we can just check if caller is picking up the same operation if not then indicate the failure.

just a question you know.

Reply:
I consider the action of *trying* to set an instance to be faulty when it's already preset.

Although...  Hmm, I have to look at this again with that perspective in mind.